Makefile

```diff
@@ -258,7 +259,12 @@ artifacts: kustomize
 	$(KUSTOMIZE) build config/dev -o artifacts/manifests-dev.yaml
 	$(KUSTOMIZE) build config/prometheus -o artifacts/prometheus.yaml
 	@$(call clean-manifests)
-	cp -r charts artifacts/charts
+	# Update the image tag and policy
+	yq  e  '.controllerManager.manager.image.repository = "$(IMAGE_REPO)" | .controllerManager.manager.image.tag = "$(GIT_TAG)" | .controllerManager.manager.image.pullPolicy = "IfNotPresent"' -i charts/kueue/values.yaml
+	# create the package. TODO: consider signing it
+	helm package --version $(GIT_TAG) --app-version $(GIT_TAG) charts/kueue -d artifacts/
+	# Revert the image changes
+	yq  e  '.controllerManager.manager.image.repository = "$(STAGING_IMAGE_REGISTRY)/$(IMAGE_NAME)" | .controllerManager.manager.image.tag = "main" | .controllerManager.manager.image.pullPolicy = "Always"' -i charts/kueue/values.yaml
```

Comment:
make this part of clean-manifiests

Reply:
let's keep it separate, this is only useful here,  clean manifests is used on a lot more targets